File: igrant_org_sdk/src/main/java/io/igrant/igrant_org_sdk/models/Consent/ConsentStatusRequest.java
package io.igrant.igrant_org_sdk.models.Consent;

public class ConsentStatusRequest {

    private String Consented;

    private int Days;

    public String getConsented() {
        return Consented;
    }

    public void setConsented(String consented) {
        Consented = consented;
    }

    public int getDays() {
        return Days;
    }

    public void setDays(int days) {
        Days = days;
    }
}

File: privacy-dashboard/src/main/java/com/github/privacydashboard/models/consent/ConsentStatusRequest.kt
package com.github.privacydashboard.models.consent

import com.google.gson.annotations.SerializedName

data class ConsentStatusRequest(
    @SerializedName("Consented")
    var consented: String? = null,
    @SerializedName("Days")
    var days: Int? = 0
) {

}
